static/app/views/starfish/views/spans/spanSummaryPanel/useSpanTransactionMetrics.tsx

```diff
@@ -0,0 +1,51 @@
+import keyBy from 'lodash/keyBy';
+
+import {useQuery} from 'sentry/utils/queryClient';
+import {HOST} from 'sentry/views/starfish/utils/constants';
+import type {Span} from 'sentry/views/starfish/views/spans/spanSummaryPanel/types';
+
+const INTERVAL = 12;
+
+type Metric = {
+  p50: number;
+  spm: number;
+};
+
+export const useSpanTransactionMetrics = (
```

Comment:
I'm seeing this trend for utils as well, I'm curious why we are doing 1 file per hook vs all the query hooks in one `queries.ts` file?

Reply:
I'm not fully sold on this, but I think `queries.ts` is _fine_ except we now have like 6 `queries.ts` files, and they're hard to navigate. If I know I want to see `useSpanTransactionMetrics.tsx`, it's pretty great to just ⌘+P for that filename and open it. Good developer ergonomics, IMO

Reply:
Yeah that a good point!

If we went that route, I would probably like to see all of them in a queries folder at least, another small nit of mine is that we're mixing hook a bunch of hook specific files mixed with a bunch component files.

Reply:
Yeah, structure will come when we know the structure, I agree we need more of it!